### DISCUSSION TITLE
OpenAI parsing Nutritional data in images

### DISCUSSION THREAD

**Discussion by PeterJT** [mealie-recipes/mealie#4151] [General]
From browsing the OpenAI implementation,  it appears to my novice eye that the Image processing only handles the Ingredients and Instructions from a recipe and bypasses the nutritional content. Is this the case, and if so, would it be possible to add the nutritional scanning.

**Comment by michael-genson** [comment]
Currently it supports these fields:
- name
- description
- yield
- total time
- prep time
- perform time
- ingredients
- instructions
- notes

We should be able to add nutritional info too

**Comment by PeterJT** [comment]
That was what I thought! I've been finding it useful when the recipe site has CloudFlare protection in place and I have to resort to taking a screen capture on my PC and having the AI process the image.
